### PR TITLE
Fix style checking

### PR DESCRIPTION
**Pull request by danlamanna**
Re #118.

There are still some failures:
```
 /home/dan/projects/romanesco/romanesco/__init__.py:161: [C901] 'run' is too complex (20)
3: @utils.with_tmpdir
3: ^
3: /home/dan/projects/romanesco/romanesco/plugins/docker/executor.py:65: [C901] 'run' is too complex (17)
3: def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
3: ^
3: /home/dan/projects/romanesco/romanesco/executors/workflow.py:6: [C901] 'run' is too complex (23)
3: def run(task, inputs, outputs, task_inputs, task_outputs, validate,
3: ^
3: /home/dan/projects/romanesco/romanesco/format/__init__.py:61: [C901] 'import_converters' is too complex (18)
3: def import_converters(search_paths):
3: ^
```

I think the cyclomatic complexity rule is important, but we either have to adjust it to a different threshold (it's at 14, workflow.run is at 23), or refactor some code.

### CODE REVIEW

File: romanesco/executors/workflow.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def run(task, inputs, outputs, task_inputs, task_outputs, validate,
                 raise Exception(
                     "Output %s (%s) is not in the expected type (%s) and "
                     "format (%s)." % (name, type(script_output["data"]),
-                                      vis_input["type"], d["format"]))
+                                      vis_input["type"], script_output["format"]))
```
Comment (zachmullen) [inline romanesco/executors/workflow.py]: That's why I love static analysis :)

**Comment by zachmullen** [comment]
I think all three of the complexity problems are legitimate issues that we can and should fix.

**Comment by zachmullen** [comment]
I will refactor the `romanesco.plugins.docker.executor.run` function and push it to this topic branch.

**Comment by zachmullen** [comment]
We can also now remove `pep8` from our requirements-dev.txt file.

**Comment by danlamanna** [comment]
Once #113 is merged in, ```romanesco.format.import_converters``` has a passing complexity.

**Comment by zachmullen** [comment]
For the last two complexity findings, I think we should just add ignores and open an issue to fix them later; those two functions are pretty critical and I think refactoring them is probably out of scope for this branch. Do you need review on #113, or are we waiting on @jeffbaumes ?

**Comment by danlamanna** [comment]
@zachmullen Sounds good. This PR is only failing on the test for #113, which is now ready for review.

**Comment by danlamanna** [comment]
Resolving conflicts with master.

**Comment by danlamanna** [comment]
@zachmullen This look good to you?

**Comment by zachmullen** [comment]
Lgtm

**Pull request merged**